enos/modules/run_workloads/jobs/docker-batch.nomad.hcl

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+variable "alloc_count" {
+  type    = number
+  default = 1
+}
+
+job "batch-docker" {
+  type = "batch"
+
+  group "batch-docker" {
+    count = var.alloc_count
+
+    task "batch-docker" {
+      driver = "docker"
+
+      config {
+        image   = "alpine:latest"
+        command = "sh"
+        args    = ["-c", "while true; do sleep 30000; done"]
```

Comment:
Batch workloads are a little tricky -- we probably want to make sure that they can complete and not get rescheduled by client/server restarts, rather than having them wait forever. But that'll require some new assertion logic, so let's come back to that.